float32 m_temp # C
float32 d_temp # C
float32 Voltage # V
float32 Current # A
float32 Speed # rad/s
float32 Position # rad